tests/typecheck/views/test_function_based_views.yml

```diff
@@ -1,5 +1,5 @@
--   case: http_response
-    main: |
+- case: http_response
```

Comment:
I prefer to not have style-only changes, clean diff and correct history are more important to me :)

Reply:
Sorry, my editor was auto-formatting the file and I forgot to revert that bit. I'll fix it.